autogluon/training/docker/0.6/py3/Dockerfile.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,62 @@
+{
+  "torch": [
```

Comment:
I believe this exception is only applicable to the AWS PT 1.12.1 whl, not the PT 1.12.1 whl on pypi or conda.

Please double check that this CVE is actually not applicable to the PT whl installed in this AG 0.6 DLC image.

Reply:
Addressed the exception via switching to PT base images built by AWS.